opentracing-api/src/main/java/io/opentracing/ActiveSpan.java

```diff
@@ -79,6 +84,8 @@
      */
     Continuation capture();
 
+    Span wrapped();
```

Comment:
@tylerbenson thanks for pointing this out... I meant to remove this when I changed ActiveSpan to be a subclass of Span. Though having slept on it I think there might be another (better) way: leave the class hierarchy as it was, then pass the actual `Span` instance into the Observer. Calling `finish()` on an ActiveSpan is a little weird since `deactivate()` seems like it should never happen after `finish()`.